app/views/admin/settings/forms/_rate_limit.html.erb

```diff
@@ -63,12 +63,12 @@
             <div class="crayons-field">
               <%= admin_config_label platform %>
               <p class="crayons-field__description">
-              <%= platform.capitalize %> username
+              <%= platform == "mastodon" ? "Mastodon complete URL including \"https://\"" : "#{platform.capitalize} username" %>
               </p>
               <%= social_media_field.text_field platform,
                                                 class: "crayons-textfield",
                                                 value: Settings::General.social_media_handles[platform],
-                                                placeholder: "" %>
+                                                placeholder: platform == "mastodon" ? "https://fosstodon.org/@thepracticaldev" : "" %>
```

Comment:
Mastodon is different so it requires a full URL to work

Reply:
Not blocking, but just curious: I was initially quite confused why this PR was making changes in what seemed to be an unrelated area, until I dug a bit and realized what was going on -- would it be worth separating the social media settings from the `rate_limit` partial?

Reply:
Makes sense to me too, great suggestion @jaw6 💯